.travis.yml

```diff
@@ -5,11 +5,11 @@ script: sbt ^^$TRAVIS_SBT_VERSION scripted
 matrix:
   include:
     - env: TRAVIS_SBT_VERSION="0.13.18"
-    - env: TRAVIS_SBT_VERSION="1.2.8"
+    - env: TRAVIS_SBT_VERSION="1.3.0"
 
 before_cache:
   - find $HOME/.sbt -name "*.lock" | xargs rm
-  - find $HOME/.ivy2 -name "ivydata-*.properties" | xargs rm
+  - find $HOME/.ivy2 -name "ivydata-*.properties" | xargs --no-run-if-empty rm
```

Comment:
Why `--no-run-if-empty` switch was added? Were there any problems?

Reply:
`sbt 1.3.x` uses Courseier to resolve dependencies.
I think that `ivydata-*.properties` is not made in `sbt 1.3.x`.